samples/nrf_desktop/src/modules/ble_adv.c

```diff
@@ -29,20 +29,62 @@ static const struct bt_data ad[] = {
 			  0x12, 0x18,	/* HID Service */
 #endif
 			  0x0f, 0x18),	/* Battery Service */
+#if CONFIG_DESKTOP_BLE_SWIFT_PAIR
```

Comment:
You can consider moving the Vendor Specific AD at the end of this array. When implementing the following requirement for Swift Pair:
"Remove the vendor section >30 seconds before exiting pairing mode." (mentioned by @jhn-nordic)
you would be able to switch to normal advertising without wasting the memory for the second AD data buffer

Reply:
Done. When I was putting the vendor section at the end of ad data, the advertising was unable to start - we overflown 31 bytes which are allowed. Because of that, I switched to automatic device name management in advertising and scan response data by using `BT_LE_ADV_OPT_USE_NAME`.